build-tools/automation/azure-pipelines.yaml

```diff
@@ -160,14 +160,11 @@ stages:
         downloadPath: $(System.DefaultWorkingDirectory)
 
     - task: MSBuild@1
-      displayName: build xaprepare
+      displayName: msbuild Xamarin.Android /t:Prepare
       inputs:
-        solution: build-tools\xaprepare\xaprepare.sln
+        solution: Xamarin.Android.sln
         configuration: $(XA.Build.Configuration)
-        msbuildArguments: $(AutoProvisionArgs) $(AndroidTargetAbiArgs) /t:"Restore;Build"
-
-    - script: build-tools\xaprepare\xaprepare\bin\$(XA.Build.Configuration)\xaprepare.exe -v:d --no-emoji --run-mode=CI -a --bundle-path="$(System.DefaultWorkingDirectory)"
-      displayName: run xaprepare
+        msbuildArguments: $(AutoProvisionArgs) $(AndroidTargetAbiArgs) /t:Prepare /bl:$(System.DefaultWorkingDirectory)\bin\Build$(XA.Build.Configuration)\msbuild-prepare.binlog /p:BundleRootPath=$(System.DefaultWorkingDirectory)
```

Comment:
I think we'll need to add something like this
```xml
<PropertyGroup>
  <XAPrepareBundleArgs Condition=" '$(BundleRootPath)' != '' ">--bundle-path='$(BundleRootPath)</XAPrepareBundleArgs >
</PropertyGroup>
<Exec Command="$(_XAPrepareExe) $(_XAPrepareStandardArgs) $(XAPrepareBundleArgs)" WorkingDirectory="$(_TopDir)" />
```
to https://github.com/xamarin/xamarin-android/blob/master/build-tools/scripts/PrepareWindows.targets#L18

Though the alternate bundle path recognition appears to have somehow  broken after the xaprepare PR was merged...